recipes-support/mavlink-router/files/main.conf

```diff
@@ -0,0 +1,15 @@
+[General]
+tcp-server-port=5760
+report-stats=false
+
+[UartEndpoint uart]
+Device = /dev/ttyS1
+Baud = 460800
+
+[UdpEndpoint wifi-ap]
```

Comment:
I would name this "wifi" only. So people switching to client mode can simply drop a file in conf.d to update the address. Would be nice not to need the address, and simply pass the iface to be used, making mavlink-router to broadcast on all ip broadcast addresses of that interface. But that is something that needs to be implemented on mavlink-router